types/underscore/index.d.ts

```diff
@@ -998,26 +1003,32 @@ declare module _ {
         ): number;
 
         /**
-        * A function to create flexibly-numbered lists of integers, handy for each and map loops. start, if omitted,
-        * defaults to 0; step defaults to 1. Returns a list of integers from start to stop, incremented (or decremented)
-        * by step, exclusive.
-        * @param start Start here.
-        * @param stop Stop here.
-        * @param step The number to count up by each iteration, optional, default = 1.
-        * @return Array of numbers from `start` to `stop` with increments of `step`.
-        **/
-
+         * A function to create flexibly-numbered lists of integers, handy for
+         * `each` and `map` loops. Returns a list of integers from `start`
+         * (inclusive) to `stop` (exclusive), incremented (or decremented) by
+         * `step`, exclusive. Note that ranges that `stop` before they `start`
+         * are considered to be zero-length instead of negative - if you'd like
+         * a negative range, use a negative `step`.
```

Comment:
Hmm, that's how the current documentation reads.

![image](https://user-images.githubusercontent.com/8692126/88113031-d857f580-cb65-11ea-9578-a0b7d78d079d.png)

I admittedly wasn't super sure of this myself but had interpreted this as possibly helping to indicate that if you have a step that isn't 1 or -1 that would step past stop it would still exclude the value.

Reply:
Ah, I see. I think that needs to change in the official documentation. Here: https://github.com/jashkenas/underscore/pull/2864.

Reply:
Updated!